src/targets/mas_steekproef_zavelberg/_targets_r/targets/steekproeftrekking.R

```diff
@@ -4,7 +4,7 @@ list(
     allocatie(steekproefkader = steekproefkader_finaal,
               min_samplesize = 200,
               target_samplesize = 9887,
```

Comment:
Maybe better to calculate '9887' in a separate target and then pass it to here?

Reply:
Yes, sounds good. I will do this in a separate branch.